packages/relay/src/ServerConfigParams.ts

```diff
@@ -369,6 +371,20 @@ export function validateBalanceParams (config: ServerConfigParams): void {
   }
 }
 
+export async function validateHub (hubFile: string, hubAddress: string, managerAddress: string, workerAddress: string, contractInteractor: ContractInteractor): Promise<void> {
+  const managerNonce = await contractInteractor.getTransactionCount(managerAddress, 'pending')
```

Comment:
we don't care about manager: it is perfectly ok to have the same manager (with nonce!=0) for multiple relayhubs (actually, it is even expected, as we want to reuse the stake)

Reply:
This check doesn't prevent the user from using the same manager with a different hub, it's being used to know if the hub lock file should be overwritten silently, or the user has to manually remove it since there has been activity on a different hub, and we currently do not have any way to know if that makes the relayer penalizable.